source/docs/03-metadata.xml

```diff
@@ -1308,6 +1310,7 @@
                   </specList>
                </p>
                <p>The content of the <gi scheme="MEI">item</gi> is quite similar to <gi scheme="MEI">manifestation</gi> with a few omissions:</p>
+               <p>Please note that in case of handwritten sources the <att>singleton</att> attribute (inspired by <ref target="https://repository.ifla.org/handle/123456789/659">FRBRoo</ref>) is available in <gi scheme="MEI">manifestation</gi> to indicate a unique physical object, because <gi scheme="MEI">item</gi> and <gi scheme="MEI">manifestation</gi> are essentially the same.</p>
```

Comment:
Well, the documentation is not as precise as the scope note of Manifestation Singleton in [FRBRoo](https://www.ifla.org/wp-content/uploads/2019/05/assets/cataloguing/FRBRoo/frbroo_v_2.4.pdf), which is what inspired it in MEI, but - at least - I understand them to be conceptually the same. 
The scope note precisely says:
> This class comprises physical objects that each carry an instance of F2 Expression,
and that were produced as unique objects, with no siblings intended in the course of
their production. It should be noted that if all but one copy of a given publication
are destroyed, then that copy does not become an instance of F4 Manifestation
Singleton, because it was produced together with sibling copies, even though it now
happens to be unique. Examples of instances of F4 Manifestation Singleton include
manuscripts, preparatory sketches and the final clean draft sent by an author or a
composer to a publisher.

So I think both cases you described @ahankinson do not satisfy the "intent" criterion and are thus not examples of (true) manifestation singletons.

Reply:
We have many cases in RISM where a library holds a very special copy of a print -- for example, one with composer handwritten annotations, or annotations from a famous conductor.

They often mis-catalogue these sources as "manuscript" since, to them, the manuscript component of these prints is what makes them worth cataloguing; not the print.

I say this only to suggest that the wording as it is, is ambiguous about this. The wording from the FRBRoo scope note is more precise, in that it specifically states **"produced as unique objects, with no siblings intended in the course of
their production."** That bit is important, I think, for understanding what a 'manifestation singleton' is. The second sentence is even clearer on the case of unica. 

I think they put these in the scope notes because it's precisely the type of confusion that would arise here. 

So I might suggest adding something similar to the MEI guidelines -- and probably also a link to the definition in FRBRoo, because that is helpful -- to ensure people understand the specific circumstances where it should be used.

Also -- I'm not sure what happens with multiple copies of the same manuscript... is it a singleton or not?